Source/WebCore/dom/Element.h

```diff
@@ -222,6 +222,31 @@ class Element : public ContainerNode {
     enum class TopLayerElementType : bool { Other, Popover };
     HTMLElement* topmostPopoverAncestor(TopLayerElementType topLayerType);
 
+#if ENABLE(ARIA_IDL_REFLECTION)
+    // aria-current uses [ReflectSetter] rather than [Reflect, Enumerated] due to complex reflection requirements.
+    const AtomString& ariaCurrent() const;
+
+    enum class AriaAtomicValues { True, False };
+    enum class AriaAutoCompleteValues { Inline, List, Both, None };
+    enum class AriaBusyValues { True, False };
+    enum class AriaCheckedValues { True, False, Mixed };
+    enum class AriaDisabledValues { True, False };
+    enum class AriaExpandedValues { True, False };
+    enum class AriaHasPopupValues { Grid, Tree, Listbox, Dialog, Menu, True, False };
+    enum class AriaHiddenValues { True, False };
+    enum class AriaInvalidValues { Grammar, Spelling, True, False };
+    enum class AriaLiveValues { Polite, Assertive, Off };
+    enum class AriaModalValues { True, False };
+    enum class AriaMultiLineValues { True, False };
+    enum class AriaMultiSelectableValues { True, False };
+    enum class AriaOrientationValues { Horizontal, Vertical };
+    enum class AriaPressedValues { Mixed, True, False };
+    enum class AriaReadOnlyValues { True, False };
+    enum class AriaRequiredValues { True, False };
+    enum class AriaSelectedValues { True, False };
+    enum class AriaSortValues { Ascending, Descending, Other, None };
```

Comment:
We should give types to these, as otherwise they'll default to the too-large value of `int`. Let's use `bool` for the two-value enums, and `uint8_t` for the three-value enums.

`enum class AriaAtomicValues : bool { True, False };`
`enum class AriaAutoCompleteValues : uint8_t { Inline, List, Both, None };`